### ISSUE TITLE
Error on pod install - "unknown option single-branch"

### ISSUE THREAD

**Issue by tygermeow** [samkaufman/CurrencyRequest#2]
Not sure why this install is failing with an error "unknown option single-branch"

matsu-macbook:currencyconverter matsuthornton$ pod install
Updating local specs repositories
Analyzing dependencies
Downloading dependencies
Installing CurrencyRequest (0.1.0)

[!] Error installing CurrencyRequest
[!] /opt/local/bin/git clone https://github.com/samkaufman/CurrencyRequest.git /var/folders/h7/tbs8lkbj32j8713z4f1_dc1w0000gn/T/d20151228-66615-piiqnq --single-branch --depth 1 --branch 0.1.0

error: unknown option `single-branch'

Podfile:
# Uncomment this line to define a global platform for your project
# platform :ios, '7.0'
# Uncomment this line if you're using Swift
# use_frameworks!

target 'CurrencyConverter' do

pod 'CurrencyRequest','~> 0.1'
# pod 'CurrencyRequest',':git => "https://github.com/samkaufman/CurrencyRequest.git", :tag => s.version.to_s'
# pod "CurrencyRequest"

end
# target 'CurrencyConverterTests' do
# end

**Comment by djp3** [comment]
If you are asking as part of the Coursera course, let me encourage you to visit 
http://discourse.djp3.net/ http://discourse.djp3.net/

---

Associate Professor Donald J. Patterson, http://www.djp3.net http://www.djp3.net/
 http://www.djp3.net/

\

**Comment by tygermeow** [comment]
Thank you for your response.  I found out that my computer had an old version of git 1.7.3 and when I upgraded it using the install package for 2.6.3 it did not update to the correct location.

They added the support for --single-branch in version 1.7.10

I added I added:
export PATH=/usr/local/bin:$PATH
to my ~/.bashrc file

then I had to run 
source ~/.bashrc from the directory I wanted to install pods into and now it works.

Thank you for the help!!
